easybuild/easyconfigs/p/Python/Python-3.9.5-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,441 @@
+        'modulename': 'sphinxcontrib.devhelp',
+        'checksums': ['ff7f1afa7b9642e7060379360a67e9c41e8f3121f2ce9164266f61b9f4b338e4'],
+    }),
+    ('sphinxcontrib-htmlhelp', '1.0.3', {
+        'modulename': 'sphinxcontrib.htmlhelp',
+        'checksums': ['e8f5bb7e31b2dbb25b9cc435c8ab7a79787ebf7f906155729338f3156d93659b'],
+    }),
+    ('sphinxcontrib-jsmath', '1.0.1', {
+        'modulename': 'sphinxcontrib.jsmath',
+        'checksums': ['a9925e4a4587247ed2191a22df5f6970656cb8ca2bd6284309578f2153e0c4b8'],
+    }),
+    ('sphinxcontrib-qthelp', '1.0.3', {
+        'modulename': 'sphinxcontrib.qthelp',
+        'checksums': ['4c33767ee058b70dba89a6fc5c1892c0d57a54be67ddd3e7875a18d14cba5a72'],
+    }),
+    ('sphinxcontrib-serializinghtml', '1.1.4', {
+        'modulename': 'sphinxcontrib.serializinghtml',
+        'checksums': ['eaa0eccc86e982a9b939b2b82d12cc5d013385ba5eadcc7e4fed23f4405f77bc'],
+    }),
+    ('Sphinx', '4.0.0', {
+        'checksums': ['b246ebd74f5fb966d7e90086bbda5ed74ee4d30b4c3cbefddc1fb5210aa317c7'],
+    }),
+    ('sphinx-bootstrap-theme', '0.7.1', {
+        'checksums': ['571e43ccb76d4c6c06576aa24a826b6ebc7adac45a5b54985200128806279d08'],
+    }),
+    ('colorama', '0.4.4', {
+        'checksums': ['5941b2b48a20143d2267e95b1c2a7603ce057ee39fd88e7329b0c292aa16869b'],
+    }),
+    ('click', '7.1.2', {
+        'checksums': ['d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a'],
+    }),
+    ('psutil', '5.8.0', {
+        'checksums': ['0c9ccb99ab76025f2f0bbecf341d4656e9c1351db8cc8a03ccd62e318ab4b5c6'],
+    }),
+    ('future', '0.18.2', {
+        'checksums': ['b1bead90b70cf6ec3f0710ae53a525360fa360d306a86583adc6bf83a4db537d'],
+    }),
+    ('sortedcontainers', '2.3.0', {
+        'checksums': ['59cc937650cf60d677c16775597c89a960658a09cf7c1a668f86e1e4464b10a1'],
+    }),
+    ('intervaltree', '3.1.0', {
+        'checksums': ['902b1b88936918f9b2a19e0e5eb7ccb430ae45cde4f39ea4b36932920d33952d'],
+    }),
+    ('pytoml', '0.1.21', {
+        'checksums': ['8eecf7c8d0adcff3b375b09fe403407aa9b645c499e5ab8cac670ac4a35f61e7'],
+    }),
+    ('flit-core', '3.2.0', {
+        'source_tmpl': 'flit_core-%(version)s.tar.gz',
+        'checksums': ['ff87f25c5dbc24ef30ea334074e35030e4885e4c5de3bf4e21f15746f6d99431'],
+    }),
+    ('flit', '3.2.0', {
+        'checksums': ['592464c9268bbacec9bc67b5a3ae62e6e090aeec1563e69501df338a1728e551'],
+    }),
+    ('regex', '2021.4.4', {
+        'checksums': ['52ba3d3f9b942c49d7e4bc105bb28551c44065f139a65062ab7912bef10c9afb'],
+    }),
+    ('intreehooks', '1.0', {
+        'checksums': ['87e600d3b16b97ed219c078681260639e77ef5a17c0e0dbdd5a302f99b4e34e1'],
+    }),
+    ('crashtest', '0.3.1', {
+        'use_pip': False,
+        'checksums': ['42ca7b6ce88b6c7433e2ce47ea884e91ec93104a4b754998be498a8e6c3d37dd'],
+    }),
+    ('pylev', '1.3.0', {
+        'checksums': ['063910098161199b81e453025653ec53556c1be7165a9b7c50be2f4d57eae1c3'],
+    }),
+    ('pastel', '0.2.1', {
```

Comment:
Can you use the whl and pip: https://github.com/easybuilders/easybuild-easyconfigs/pull/12650/files#diff-62174d1a70710afde8442e36d7bd56cb93bf98a80eaa010139cd3770c5f9df49R337